pkg/options/options.go

```diff
@@ -129,4 +130,5 @@ func (s *KubeRouterConfig) AddFlags(fs *pflag.FlagSet) {
 	// 	"Password that cluster-node BGP servers will use to authenticate one another when \"--nodes-full-mesh\" is set.")
 	fs.StringVarP(&s.VLevel, "v", "v", "0", "log level for V logs")
 	fs.Uint16Var(&s.HealthPort, "health-port", 20244, "Health check port, 0 = Disabled")
+	fs.BoolVar(&s.PathPrepend, "path-prepend", false, "tktk")
```

Comment:
Not sure we need this, we should just turn on path prepend if it's found in the annotations.